components/Feedback.tsx

```diff
@@ -0,0 +1,22 @@
+import { useTranslation } from 'next-i18next'
+import { Trans } from 'react-i18next'
+
+export const Feedback: React.FC = () => {
+  const { t } = useTranslation('common')
+
+  return (
+    <div className="feedback">
+      <div className="feedback-content">
+        <div className="feedback-title">{t('feedback.title')}</div>
+        <div className="feedback-message">
+          {
+            // TODO correct feedback link?
```

Comment:
Would be good for this to be just a touch more specific, so another engineer could pick it up rapidly if need be, e.g. "Is this correct i18nKey attribute value?"  or "TODO: Insert correct href link." (took me a second to realize what this meant, for instance). 

Absolutely not a blocking comment, just a grumpy old programmer comment, someone who has written many similar TODOs. :)

Reply:
honestly I owe a pass on finding my TODOs and confirming tickets exist - I'll do that now.